checker/check_runner.go

```diff
@@ -101,7 +101,9 @@ func (r *Runner) Run(ctx context.Context, f CheckFn) CheckResult {
 		break
 	}
 	res.Details2 = l.messages2
-
+	for _, d := range l.messages2 {
```

Comment:
this will disappear from the PR once https://github.com/ossf/scorecard/pull/841 is merged.